app/main.py

```diff
@@ -2,5 +2,15 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_data = {}
+
+    def wrapper(*args) -> Callable:
```

Comment:
You can return not only callable, because your functions can return any type

Reply:
Many thanks for your correction. I got it. I understood why exactly so and for what